met-web/src/components/publicDashboard/Dashboard.tsx

```diff
@@ -52,7 +60,7 @@ const Dashboard = () => {
             })
             .catch((error) => {
                 // Error handling
-                console.error('Error generating dashboard PDF:', error);
+                console.error(translate('dashboard.consoleError.pdfError'), error);
```

Comment:
Console errors don't need translation so no need to handle these. We're making the assumption that our team's developers work in English

Reply:
Removed the translation for console error. Thanks.